maxim/logger/livekit/module_functions.py

```
@@ -0,0 +1,109 @@
+def instrument_execute_tools_task():
+    """
+    Instrument the execute_tools_task function in livekit.agents.voice.generation
+    """
+    try:
+        import importlib
+
+        module = importlib.import_module("livekit.agents.voice.generation")
+
+        # Correct function name check
+        if hasattr(module, "_execute_tools_task"):
+            original_func = getattr(module, "_execute_tools_task")
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)